realm/realm-library/src/syncIntegrationTest/java/io/realm/SSLConfigurationTests.java

```diff
@@ -65,12 +65,12 @@ public void trustedRootCA() throws InterruptedException {
         SystemClock.sleep(TimeUnit.SECONDS.toMillis(2));  // FIXME: Replace with Sync Progress Notifications once available.
         realm.close();
         user.logout();
-        Realm.deleteRealm(configOld);
 
         // 2. Local state should now be completely reset. Open the Realm again with a new configuration which should
         // download the uploaded changes.
         user = SyncUser.login(SyncCredentials.usernamePassword(username, password), Constants.AUTH_URL);
         SyncConfiguration config = configurationFactory.createSyncConfigurationBuilder(user, Constants.USER_REALM_SECURE)
+                .name("newRealm")
```

Comment:
These tests are disabled for now, https://github.com/realm/ros/issues/512 any reason to add the name?

Reply:
because of we sadly don't have a reliable way to know when the sync client thread will close the Realm instance.

https://github.com/realm/realm-java/pull/5417/files#diff-c4f8c4a7fed4b6347565a6e587f5da61L68